src/foam/nanos/auth/Group.js

```diff
@@ -38,5 +44,25 @@ foam.CLASS({
       documentation: 'Custom authentication settings for this group.'
     }
     */
+  ],
+  methods: [
+    {
+      name: 'implies',
+      javaReturns: 'Boolean',
+      args: [
+        {
+          name: 'name',
+          javaType: 'String'
+        }
+      ],
+      javaCode:
+        'if (getPermissions() == null) {return false;} \n' +
+        'for (int i = 0; i < permissions_.length; i++) { \n' +
+        '  if (permissions_[i].getId().equals(name)) { \n' +
```

Comment:
Don't use equality, create a java.security.Permission and call its implies() method. This adds wildcard support.